fluentd/configs.d/filter/cloud_init.conf

```diff
@@ -0,0 +1,5 @@
+<filter journal>
+  @type grep
+  # drop messages from cloud-init - they are not logged to /var/log/messages by default
+  exclude1 _COMM ^cloud-init$
```

Comment:
is there really a need to remove this? it still goes to the logs which admin can see.